tools/installer/installer.iss

```diff
@@ -61,97 +64,117 @@ begin
   UUIDPath := Format('%s\planetarium\.installer_mixpanel_uuid', [ExpandConstant('{localappdata}')]);
   if (FileExists(UUIDPath)) then
   begin
-    LoadStringFromFile(UUIDPath, LoadedUUID)
+    LoadStringFromFile(UUIDPath, LoadedUUID);
     Result := LoadedUUID;
   end else begin
     Result := GenerateUUID();
     SaveStringToFile(UUIDPath, Result, False);
   end
 end;
 
+function InitializeSetup(): Boolean;
+begin
+  UUID := UseUUID();
+  Result := True;
+end;
+
 procedure MixpanelTrack(Event, UUID: String);
 var
   WinHttpReq: Variant;
 begin
   WinHttpReq := CreateOleObject('WinHttp.WinHttpRequest.5.1');
   WinHttpReq.Open('POST', 'https://api.mixpanel.com/track', False);
   WinHttpReq.SetRequestHeader('Content-Type', 'application/x-www-form-urlencoded');
-  WinHttpReq.Send(Format('data={"event":"%s","properties":{"token":"80a1e14b57d050536185c7459d45195a","action":"start","distinct_id":"%s"}}', [Event, UUID]));
+  WinHttpReq.Send(Format('data={"event":"%s","properties":{"token":"80a1e14b57d050536185c7459d45195a","distinct_id":"%s"}}', [Event, UUID]));
   if WinHttpReq.ResponseText = 1 then begin
     Log('Mixpanel request success.');
   end else begin
     Log('Mixpanel request failed. ' + WinHttpReq.ResponseText);
   end;
 end;
 
-procedure CurStepChanged(CurStep: TSetupStep);
+procedure SendUUID(Event, UUID: String);
 var
-  UUID: String;
+  WinHttpReq: Variant;
+begin
+  WinHttpReq := CreateOleObject('WinHttp.WinHttpRequest.5.1');
+  WinHttpReq.Open('POST', 'https://planetariumhq.slack.com/services/hooks/slackbot?token=4hBLriaHECDGHlNNbOnwjkfk&channel=%239c-installer', False);
```

Comment:
It seems to contain the planetariumhq team Slack bot's token. Is it okay to be shown in public?

Reply:
I agree, I'll remove this part since the UUID issue has been resolved.